core/src/avm2/verify.rs

```diff
@@ -585,25 +588,31 @@ pub fn verify_method<'gc>(
             | AbcOp::Jump { offset } => {
                 let adjusted_result = adjust_jump_to_idx(i, *offset, true)?;
                 *offset = adjusted_result.1;
-                if let Some(jump_sources) = potential_jump_targets.get_mut(&adjusted_result.0) {
-                    if let JumpSources::Known(sources) = jump_sources {
-                        sources.push(i);
-                    }
+                if let Some(sources) = potential_jump_targets.get_mut(&adjusted_result.0) {
+                    sources.push(JumpSource::JumpFrom(i));
                 } else {
-                    potential_jump_targets.insert(adjusted_result.0, JumpSources::Known(vec![i]));
+                    potential_jump_targets.insert(adjusted_result.0, vec![JumpSource::JumpFrom(i)]);
                 }
             }
             AbcOp::LookupSwitch(ref mut lookup_switch) => {
-                // TODO: Add i to possible sources, like in the branch ops
-
                 let adjusted_default = adjust_jump_to_idx(i, lookup_switch.default_offset, false)?;
                 lookup_switch.default_offset = adjusted_default.1;
-                potential_jump_targets.insert(adjusted_default.0, JumpSources::Unknown);
```

Comment:
do you remember why it was `Unknown` before?

Reply:
I didn't know of the right way to implement it before